src/components/Textarea/docs/Textarea.stories.mdx

```diff
@@ -0,0 +1,58 @@
+import { ControlledTextarea } from './ControlledTextarea';
+import { Textarea } from '../Textarea';
+import { ItemWrapper } from '../../../../docs/components/ItemWrapper';
+import { StyledSystemLinks } from '../../../docs/StyledSystemLinks';
+import { TextareaPropsTable } from './TextareaPropsTable';
+import { Canvas, Story, ArgsTable, Meta } from '@storybook/addon-docs';
+
+# Textarea
+
+<Meta title="Components/Textarea" component={Textarea} />
+
+export const Template = args => <Textarea {...args} />;
+
+<ArgsTable of={Textarea} />
+<StyledSystemLinks component="Textarea" supportedProps={['margin', 'height', 'width']} />
+
+## Properties
+
+<ArgsTable of={Textarea} />
+
+## Playground
+
+<Canvas>
+    <Story
+        name="Basic"
+        args={{
+            placeholder: 'user@free-now.com',
+            label: 'E-Mail Address',
+            resize: 'vertical'
+        }}
+    >
+        {Template.bind({})}
+    </Story>
+</Canvas>
+
+## Appearance
+
+### Default
+
+<Textarea label="Default" />
```

Comment:
Can we take the Storybook approach here and use `<Canvas>` and `<Story>` components?

Example: src/components/Password/docs/Password.stories.mdx in https://github.com/freenowtech/wave/pull/319/files#diff-d446ba0b705dc09e92956a803cb2dfd54917947ccea20dfa0635c3280ef0a336